configs/next/packages/gcc/sources

```diff
@@ -22,8 +22,8 @@
 ATSRC_PACKAGE_NAME="GCC (GNU Compiler Collection)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_0="GNU Standard C++ Library v3 (Libstdc++-v3)"
 ATSRC_PACKAGE_SUBPACKAGE_NAME_1="GNU Libgomp"
-ATSRC_PACKAGE_VER=9.0.0
-ATSRC_PACKAGE_REV=4fd240469362
+ATSRC_PACKAGE_VER=9.0.1
+ATSRC_PACKAGE_REV=bf4d9552d7c3
```

Comment:
OK (since GCC builds fail when there's a version mismatch, I'd suggest to wait for the new build results).

```
$ git show bf4d9552d7c3:gcc/BASE-VER
9.0.1
```